spec/lib/miq_automation_engine/models/miq_ae_field_spec.rb

```diff
@@ -117,37 +117,37 @@
       f2.save!.should be_true
       @c1.reload
       fields = @c1.ae_fields
-      fields.length.should == 1
-      fields[0].priority.should == 2
+      fields.length.should be == 1
+      fields[0].priority.should be == 2
```

Comment:
Prefer
`.should eq(2)` over `be == 2`
or even better just switch to the expect syntax:
`expect(blah.priority).to eq(2)`